app/lib/server/functions/deleteUser.js

```diff
@@ -99,7 +106,6 @@ export const deleteUser = function(userId) {
 
 	Users.removeById(userId); // Remove user from users database
 
-	// Refresh the peers list
-	const { peer: { domain: localPeerDomain } } = getConfig();
-	FederationPeers.refreshPeers(localPeerDomain);
+	// Refresh the servers list
+	FederationServers.refreshServers();
```

Comment:
although this was already here, it should be better if we have a `afterDeleteUser` callback.